db_scripts/mosip_idpbinding/ddl/idpbinding-id_token_mapping.sql

```diff
@@ -0,0 +1,29 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-- -------------------------------------------------------------------------------------------------
+-- Database Name: mosip_idpbinding
+-- Table Name : id_token_mapping
+-- Purpose    : id_token_mapping: Table to store Id Hash and respective PSU Tokens.
+--           
+-- Create By   	: Sowmya B
+-- Created Date	: Nov-2022
+-- 
+-- Modified Date        Modified By         Comments / Remarks
+-- ------------------------------------------------------------------------------------------
+-- ------------------------------------------------------------------------------------------
+
+-- object: id_token_mapping.id_token_mapping | type: TABLE --
+-- DROP TABLE IF EXISTS id_token_mapping .id_token_mapping  CASCADE;
+CREATE TABLE id_token_mapping (
+	id_hash character varying(50) NOT NULL,
+	psu_token character varying(256) NOT NULL,
+	CONSTRAINT pk_id_token_mapping PRIMARY KEY (id_hash),
+	CONSTRAINT uk_id_token_mapping UNIQUE (psu_token)
```

Comment:
Here we should not apply unique key constraint.

Reply:
Fixed